devops/rpmbuild/extrafiles/basecoin/usr/share/basecoin/key2.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
What about renaming the key2.json to test-key-2.json, just to make it very clear that they are test files.